scss/titles.scss

```diff
@@ -0,0 +1,5 @@
+.title {
```

Comment:
where's the triangle image that appears behind the title 🤔 this image will always be attached to the title component. since it's purely aesthetic we shouldn't add it in the html but we can add it here as a background or using a before pseudo element